src/test/groovy/com/avioconsulting/mule/linter/rule/property/PropertyFileNamingRuleTest.groovy

```diff
@@ -0,0 +1,28 @@
+package com.avioconsulting.mule.linter.rule.property
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.Rule
+import com.avioconsulting.mule.linter.model.RuleViolation
+import spock.lang.Specification
+
+class PropertyFileNamingRuleTest extends Specification {
+
+    @SuppressWarnings(['MethodName', 'MethodReturnTypeRequired', 'GStringExpressionWithinString'])
```

Comment:
Need more tests, should text using default pattern and custom pattern as well as both passing and failing scenarios.  Might need more sample projects to test, although they can be minimal projects with just a pom + property files.

Reply:
Should probably check more then just violation size as well in at least one test, are you returning violations with correct data?